silkworm/rpc/http/connection.hpp

```diff
@@ -60,55 +60,40 @@ class Connection : public Channel {
     //! Start the asynchronous read loop for the connection.
     Task<void> read_loop();
 
-    Task<void> write_rsp(Response& response) override;
+    Task<void> write_rsp(std::string& response) override;
     Task<void> open_stream() override;
     Task<std::size_t> write(std::string_view content) override;
     Task<void> close() override { co_return; }
 
   private:
     using AuthorizationError = std::string;
     using AuthorizationResult = tl::expected<void, AuthorizationError>;
-    AuthorizationResult is_request_authorized(const http::Request& request);
+    AuthorizationResult is_request_authorized(boost::beast::http::request<boost::beast::http::string_body>& req);
 
-    Task<void> handle_request(Request& request);
+    Task<void> handle_request(boost::beast::http::request<boost::beast::http::string_body>& req);
```

Comment:
The `req` parameter should be either `const T&` or `T&&` here

Reply:
ok, i have declared constT&